tasks/fh-build.js

```diff
@@ -366,6 +366,25 @@ module.exports = function(grunt) {
                       'shell:fh-run-array:accept']);
     } else if (this.target === 'unit') {
       grunt.task.run(['shell:fh-run-array:unit']);
+    } else if (this.target === 'testfile'){
```

Comment:
Do you think moving some of the logic inside this 'else if' block would make things easier to follow?